src/Symfony/Component/Messenger/Tests/Transport/Doctrine/ConnectionTest.php

```diff
@@ -345,4 +350,53 @@ public function testFindAll()
         $this->assertEquals('{"message":"Hi again"}', $doctrineEnvelopes[1]['body']);
         $this->assertEquals(['type' => DummyMessage::class], $doctrineEnvelopes[1]['headers']);
     }
+
+    /**
+     * @dataProvider providePlatformSql
+     */
+    public function testGeneratedSql(AbstractPlatform $platform, string $expectedSql)
+    {
+        $driverConnection = $this->createMock(DBALConnection::class);
+        $driverConnection->method('getDatabasePlatform')->willReturn($platform);
+        $driverConnection->method('createQueryBuilder')->willReturnCallback(function () use ($driverConnection) {
+            return new QueryBuilder($driverConnection);
+        });
+
+        if (interface_exists(DriverResult::class)) {
+            $result = $this->createMock(DriverResult::class);
+            $result->method('fetchAssociative')->willReturn(false);
+
+            if (class_exists(Result::class)) {
+                $result = new Result($result, $driverConnection);
+            }
+        } else {
+            $result = $this->createMock(ResultStatement::class);
+            $result->method('fetch')->willReturn(false);
+        }
+
+        $driverConnection->expects(self::once())->method('beginTransaction');
+        $driverConnection
+            ->expects(self::once())
```

Comment:
`this->once` for consistency in this file.
same above

Reply:
Done.